jobs/azure_cpi/templates/cpi.json.erb

```diff
@@ -55,6 +51,21 @@
     end
   end
 
+  if_p('azure.default_managed_identity.type') do |identity_type|
+    if identity_type == 'SystemAssigned'
+      params['cloud']['properties']['azure']['default_managed_identity'] = {
+        "type" => identity_type
+      }
+    elsif identity_type == 'UserAssigned'
+      params['cloud']['properties']['azure']['default_managed_identity'] = {
+        "type" => identity_type,
+        "user_assigned_identity_name" => p('azure.default_managed_identity.user_assigned_identity_name')
```

Comment:
probably need to confirm that `azure.default_managed_identity.user_assigned_identity_name` is not nil

Reply:
If `user_assigned_identity_name` is not specified, users will get an error `#<TemplateEvaluationContext::UnknownProperty: Can't find property 'azure.default_managed_identity.user_assigned_identity_name'>` when rendering `cpi.json`. So we don't need to check whether the property is specified.